src/application/trading-copilot/commands/get-ens-address.ts

```diff
@@ -0,0 +1,44 @@
+import { createPublicClient, http } from 'viem';
+import { mainnet } from 'viem/chains';
```

Comment:
I didn't catch this before - please use chains from shared/web3:

CHAIN.ETHEREUM is equivlent of mainnet